src/app/share/services/share.service.ts

```diff
@@ -376,21 +376,25 @@ export const getPublicShareLink = async (uuid: string, itemType: 'folder' | 'fil
   const encryptedMnemonic = aes.encrypt(mnemonic, code);
 
   try {
-    const publicSharingItemData = await shareService.createPublicSharingItem({
+    const publicSharingItemData = await createPublicSharingItem({
       encryptionAlgorithm: 'inxt-v2',
       encryptionKey: encryptedMnemonic,
       itemType,
       itemId: uuid,
+      encryptedCode: code,
```

Comment:
The code needs to be sent encrypted with the mnemonic. Please check how we were doing it on the old sharing and you will find the line where the code is encrypted with the mnemonic previously to be sent. @CandelR